config/i18n-tasks.yml

```diff
@@ -97,6 +97,7 @@ ignore_unused:
   - 'experiments.*'
   - 'event_types.*'
   - 'zxcvbn.*'
+  - 'links.cancel'
```

Comment:
Yeah this is here because I was waiting for the cancel link PR (https://github.com/18F/identity-idp/pull/860) to get merged, and I didn't want to have to remove the key only to re-add it in that PR in case this one got merged first.